secure/setting-up-openshift-pipelines-to-view-software-supply-chain-security-elements.adoc

```diff
@@ -0,0 +1,33 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="setting-up-openshift-pipelines-to-view-software-supply-chain-security-elements"]
+= Setting up Openshift Pipelines to view Software Supply Chain Security elements
+:context: setting-up-openshift-pipelines-to-view-software-supply-chain-security-elements
+
+toc::[]
+
+[role="_abstract"]
+Use the *Developer* or *Administrator* perspective to create or modify a Pipeline and view key Software Supply Chain Security elements within a project.
+
+Set up Pipelines to view:
+
+* **Project vulnerabilities**: Visual representation of identified vulnerabilities within a project.
+
+* **Software Bill of Materials (SBOMs)**: Download or view detailed listing of PipelineRun components.
+
+Additionally, PipelineRuns that meet {tekton-chains}' requirements display a signed badge next to its name. This badge indicates that the PipelineRun's execution results are cryptographically signed and stored securely, for example within an OCI image.
```

Comment:
I'd remove the apostrophe and the "'s". Also, consider "pipeline run execution results"